addon/styles/_frost-button.scss

```diff
@@ -305,6 +297,54 @@ $frost-color-button-in-line-darken-2: darken($frost-color-blue-1, 15);
       box-shadow: none;
     }
   }
+  
+  // design = 'app-bar'
+  &.app-bar {
```

Comment:
Sorry, I'm not sure I follow what that means.

> I'd prefer that this be included from the context where the design will be used

Do you mean the app-bar itself should have these styles? 

I thought we had this argument in slack and you were pitching for having all this in the button itself...

> e.g. Design is for global common layouts and the one-offs are included from the unique context

I thought one-offs were exactly what `design` was for. It appears I may be out-of-the-loop here. Sorry, I guess I shouldn't have approved this. I'll refrain next time.